modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-impl/rest-openapi.yaml

```diff
@@ -985,6 +1052,77 @@ paths:
                             schema:
                                 type: string
             tags: ["Order"]
+    /orders/{orderId}/shipment-infos:
```

Comment:
This will be `/placed-order-items/{placedOrderItemId}/placed-order-item-shipments`